sdk/storage/azure-storage-files-shares/test/share_file_client_test.cpp

```diff
@@ -680,6 +680,7 @@ namespace Azure { namespace Storage { namespace Test {
       auto destFileClient
           = m_shareClient->GetRootDirectoryClient().GetFileClient(LowercaseRandomString(10));
       auto copyOperation = destFileClient.StartCopy(fileClient.GetUrl());
+      EXPECT_NE(copyOperation.GetRawResponse(), nullptr);
```

Comment:
Why not checking some response code or anything else?

Reply:
because i'm lazy 😞